flowetl/tests/integration/test_fixuid.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Tests that fixuid works as expected
+"""
+
+
+def test_uid(docker_client, flowetl_tag):
+    """
+    test that we can run the flowetl container with a specific user
+    """
+
+    user = "1002:1003"
+    out = docker_client.containers.run(
+        f"flowminder/flowetl:{flowetl_tag}", "bash -c 'id -u'", user=user
+    )
+    assert out.decode("utf-8").strip() == "1002"
+
+
+def test_gid(docker_client, flowetl_tag):
```

Comment:
```suggestion
def test_uid_gid(docker_client, flowetl_tag):
```

Reply:
Why?

Reply:
The first checks the UID - and second checks the GID.

Reply:
Sorry, ignore me!